### ISSUE TITLE
A window switcher

### ISSUE THREAD

**Issue by zim0369** [hyprwm/Hyprland#1903]
A window similar to [wayfire's](https://www.youtube.com/watch?v=2PtNzxDsxYM&t=15s) is convenient when having to switch windows in full screen mode, where instead of having to juggle through all the windows the user can see a nice preview.

**Comment by vaxerski** [comment]
No, sorry but I'd rather leave 2007 features in the past. You can use some utils like eww or some others like iirc sfwbar to have alt-tab like panels already for a more 2023 look.
